packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -493,6 +493,39 @@ class ShaderMask extends SingleChildRenderObjectWidget {
 /// [ImageFiltered] instead. For that scenario, [ImageFiltered] is both
 /// easier to use and less expensive than [BackdropFilter].
 ///
+/// {@tool snippet}
+///
+/// This example shows how the common case of applying a [BackdropFilter] blur
+/// to a single child can be replaced with an [ImageFiltered] widget. This code
+/// is generally simpler and the performance will be improved dramatically.
+///
+/// ```dart
+///  return Stack(
```

Comment:
I would add a comment inside the dart block here pointing out explicitly that this is the inefficient implementation. To make sure that nobody accidentally copies this bad example.

Reply:
ahh good idea 😓